packages/flutter_tools/test/general.shard/base/io_test.dart

```diff
@@ -77,6 +77,7 @@ void main() {
     setExitFunctionForTests((int value) {});
 
     expect(() => exit(0), returnsNormally);
+    restoreExitFunction();
```

Comment:
Probably a bit paranoid, but should this go in a `finally {}`

Reply:
https://github.com/flutter/flutter/pull/48263